modules/gltf/gltf_document.cpp

```diff
@@ -3633,17 +3642,21 @@ Error GLTFDocument::_parse_materials(Ref<GLTFState> state) {
 				material->set_cull_mode(BaseMaterial3D::CULL_DISABLED);
 			}
 		}
-
 		if (d.has("alphaMode")) {
 			const String &am = d["alphaMode"];
+			material->set_depth_draw_mode(BaseMaterial3D::DEPTH_DRAW_ALWAYS);
```

Comment:
I think this should be a separate change. I could imagine cases that forcing depth draw mode might break.

Reply:
Removed to sync.